libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -698,8 +698,11 @@ string ABIFunctions::abiEncodingFunctionCompactStorageArray(
 					switch and(slotValue, 1)
 					case 0 {
 						// short byte array
-						mstore(pos, and(slotValue, not(0xff)))
-						ret := add(pos, <lengthPaddedShort>)
+						ret := pos		//assume empty array
```

Comment:
We usually don't use tabs / spaces to align things after the initial tabs - this will always turn out messy.

```suggestion
						ret := pos // assume empty array
```